src/main/java/com/org/trustservice/service/TrustGroupService.java

```diff
@@ -0,0 +1,204 @@
+package com.org.trustservice.service;
+
+import com.org.trustservice.dto.DepartmentTrustGroupsDTO;
+import com.org.trustservice.dto.TrustGroupDTO;
+import com.org.trustservice.dto.TrustGroupUpdateDTO;
+import com.org.trustservice.dto.TrustGroupUpdateResponseDTO;
+import com.org.trustservice.excpetion.DatabaseException;
+import com.org.trustservice.excpetion.ResourceNotFoundException;
+import com.org.trustservice.model.DepartmentTrustGroup;
+import com.org.trustservice.model.TrustGroupFlavour;
+import com.org.trustservice.model.TrustGroupPermission;
+import com.org.trustservice.repository.DepartmentTrustGroupRepository;
+import com.org.trustservice.repository.PermissionRepository;
+import com.org.trustservice.repository.TrustGroupFlavourRepository;
+import com.org.trustservice.util.TrustGroupNameComparator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+/**
+ * @author
+ */
+@Service
+public class TrustGroupService {
+
+
+    @Autowired
+    DepartmentTrustGroupRepository departmentTGRepository;
+
+    @Autowired
+    TrustGroupFlavourRepository trustGroupFlavourRepository;
+
+    @Autowired
+    PermissionRepository permissionRepository;
+
+    @Value("${google.permissions.1}")
+    private static String permission1;
+
+    @Value("${google.permissions.2}")
+    private static String permission2;
+
+    @Value("${google.permissions.3}")
+    private static String permission3;
+
+    @Value("${google.permissions.4}")
+    private static String permission4;
+
+
+    @Value("${google.role.writer}")
+    private static String writer;
+
+    @Value("${google.role.reader}")
+    private static String reader;
+
+    @Value("${google.role.commenter}")
+    private static String commenter;
+
+    @Value("${google.role.owner}")
+    private static String owner;
+
+    @Value("${google.role.unknown}")
+    private static String unknown;
+
+
+    /**
+     * This method will read data from database based on input data and form a response for UI
+     *
+     * @param orgCollabId
+     * @param departmentId
+     * @return returns TrustGroupUpdateResponseDTO  suitable for UI
+     */
+    public TrustGroupUpdateResponseDTO getAllTrustGroupsByDepartment(UUID orgCollabId, UUID departmentId) {
+        TrustGroupUpdateResponseDTO apiResponse;
```

Comment:
Incorrect scoping. Put it under try block.
Your code clearly throws the exception, so whats the point of having the return statement in the end.